lib/html_parser.dart

```diff
@@ -792,4 +798,14 @@ class StyledText extends StatelessWidget {
       ),
     );
   }
+
+  double calculateWidth(Display display, RenderContext context) {
+    if ((display == Display.BLOCK || display == Display.LIST_ITEM) && renderContext.parser.shrinkWrap != true) {
```

Comment:
if each branch returns, `else if` and `else` is not necessary. You can do something of the form:

```dart
if(condition1) {
  return 1;
}

if (condition2) {
  return 2;
}

return 3;
```